src/quiz/question.jsx

```diff
@@ -0,0 +1,67 @@
+/*This QuizH component shows the whole section number 2 of the questions */
+const QuizH = ({ arr }) => {
```

Comment:
the name is a little confusing because it renders a question, I think naming it `Question` would be better

The `arr` prop also could have a better name since it's not an array but an object with information about the question, so `question` could be a better name